src/Microsoft.DotNet.Tools.Compiler.Native/IntermediateCompilation/Linux/LinuxCppCompileStep.cs

```diff
@@ -87,10 +87,11 @@ private void InitializeArgs(NativeCompileSettings config)
             }
 
             // AppDep Libs
+            var baseAppDeplibPath = Path.Combine(config.AppDepSDKPath, "CPPSdk/ubuntu.14.04/x64");
```

Comment:
It's not like we wouldn't have that many folders anyway. Just because they are separated in to two levels doesn't inherently make it more organized :smile:. I'd still slightly prefer we align with everything else, but if this is truly temporary that's fine.

> the layout here is more C++ specific than they are nuget specific.

RIDs aren't necessarily a NuGet concept, they are just a concept for defining how Native Code is organized among the various parts of .NET (CoreFX, NuGet, dotnet-cli, etc.)